master/getting-started/kubernetes/reqs/system.md

```diff
@@ -0,0 +1,39 @@
+---
+title: System requirements
+canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/kubernetes/reqs/system'
+---
+
+{% include {{page.version}}/reqs-sys.md orch="Kubernetes" %}
+
+## Kubernetes
+
+#### Supported versions
+
+{{site.prodname}} {{page.version}} requires Kubernetes 1.8 or later.
+
+#### CNI plug-in enabled
+
+Enable CNI by passing `--network-plugin=cni` to kubelet. On kubeadm, this is
+the default.
+  
+#### Supported kube-proxy flags
+
+Do not use the following flags:
+- `--masquerade-all`
+- `--proxy-mode=userspace`
+
+Like Kubernetes, {{site.prodname}} offers beta support for `--proxy-mode=ipvs`. 
+Due to defects in earlier versions, ensure that you are on Kubernetes 1.9.3 or 
+later. Refer to [Enabling IPVS in Kubernetes](../../../usage/enabling-ipvs)
+for more details.
+
+#### IP pool configuration
```

Comment:
I think this section should probably be reworked or deleted as well.

It's not true that you should have an IP pool for your pods and another IP pool for your services. The requirement is that you use an IP pool that doesn't overlap with your service range (which, by is the case by default) and that it doesn't overlap with the ranged used for host IP addresses.

So maybe:

```
# IP address configuration

Calico requires that the IP range selected for pod IP addresses does not overlap with either the Kubernetes service cluster IP range, or the range from which host IPs are allocated. By default, Calico assigns IP addresses from the 192.168.0.0/16 IP pool, but this can be configured(link).
```

Reply:
Note that the host part of this is not specific to Kubernetes, and in general the range used for pods shouldn't overlap with any other address ranges in your network.